src/containers/ResourceOverviewPage.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import '../styles/HomeLandingPage.css';
+import '../styles/ResourceOverviewPage.css';
+import Typography from '@material-ui/core/Typography';
+import RefreshIcon from '@material-ui/icons/Refresh';
+import { IconButton } from '@material-ui/core';
+import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+import { withStyles } from '@material-ui/core/styles';
+import TeamMemberOverviewPage from './TeamMemberOverviewPage';
+import HospitalOverviewPage from './HospitalOverviewPage';
+import AmbulanceOverviewPage from './AmbulanceOverviewPage';
+import ResourceMenuTabs from '../components/common/resourceMenuTabs';
+import { Colours } from '../styles/Constants';
+
+const ResourceOverviewPage: React.FC = () => {
+  const [selectedTab, setTab] = useState(0);
+  const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
+    setTab(newValue);
+  };
+
+  const BackButton = withStyles({
+    root: {
+      marginRight: 33.5,
+      padding: 0,
+    },
+  })(IconButton);
+
+  const RefreshButton = withStyles({
+    root: {
+      marginLeft: '77%',
+      padding: 0,
+    },
+  })(IconButton);
+
+  const tabLabels = ['Team Members', 'Hospitals', 'Ambulances'];
+
+  let overview;
+
+  if (tabLabels[selectedTab] === 'Team Members') {
+    overview = <TeamMemberOverviewPage />;
+  } else if (tabLabels[selectedTab] === 'Hospitals') {
+    overview = <HospitalOverviewPage />;
+  } else if (tabLabels[selectedTab] === 'Ambulances') {
+    overview = <AmbulanceOverviewPage />;
+  }
+
+  return (
+    <div className="landing-wrapper">
+      <span className="title-bar">
+        <BackButton>
+          <ArrowBackIcon style={{ color: Colours.White }} />
+        </BackButton>
+        <Typography variant="h6">Resource Management</Typography>
+
+        <RefreshButton>
+          <RefreshIcon
+            onClick={() => {
```

Comment:
Getting this warning in console: 
`Failed prop type: Material-UI: You are providing an onClick event listener to a child of a button element.
Firefox will never trigger the event.
You should move the onClick listener to the parent button element.`

Think this means the `onClick` should be moved to the `RefreshButton` element